packages/material-ui/src/NativeSelect/NativeSelect.test.js

```diff
@@ -103,4 +103,22 @@ describe('<NativeSelect />', () => {
     expect(getByTestId('root')).to.have.class('foo');
     expect(getByTestId('root')).to.have.class('bar');
   });
+
+  it('should not add classes on the DOM element', () => {
```

Comment:
Can this be a conformance check instead? It seems like we need the same implementation in all `styled` components so it might be best to do a `expect(document.querySelector('[classes]')).to.equal(null)` to ensure it's fixed everywhere (at least for the rendered UI).

Reply:
Yeah makes sense. I will create a follow-up PR for this and ensure it pass in all components.